src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,41 @@
+.assess {
+  margin-left: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 164px;
+  align-items: flex-end;
+}
+
+.review {
```

Comment:
Don't create unnecessary blocks, they are a part of the `card` and can't be used outside it. For example BEM blocks can't have margins so you can reuse them in any place